src/client/gameannouncer.py

```diff
@@ -1,61 +1,60 @@
-from PyQt5.QtCore import QTimer
 from model.game import GameState
-
-from fa import maps
+import chat.friendtracker
```

Comment:
`from chat.friendtracker import build_friend_tracker` should work too.

Reply:
yes, yes it does.